### PR TITLE
add generic way to config based on environment variables

### PR DESCRIPTION
**Pull request by SISheogorath**
First steps for #2

### CODE REVIEW

**Comment by SISheogorath** [comment]
Mhm, I'm thinking about how useful it is to have such a powerful, but complex config toolchain.

If it's finished it's super easy to use and if we invest enough time you can build up a customized IRC networks running services, webchat and X inspircd node in less than 5 minutes. But I'm not sure if it is really helpful.

As we copy it as default config to the host disk and doing very crazy things it's not very handy for a user to customize this complex config engine inside the container if they don't want to use environment variables (and with 1.13 secrets).

I want to know how you think about it before continue and maybe finish this. Because I'm not sure if it is a big waste of time when things getting nice, but complex for newbies try to do easy modifications to the default configs.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,36 @@ Default ports of this container image:
...
+<tag value="&inspgTestEnv;">
+```
+
+Another actual example:
+
+```
+<badhost host="&inspgTestEnv;" reason="You are banned for some reason!">
+```
+
+To use it in your own mounted or backed in config inlcude `/conf/config.sh`.
```
Comment (Adam-) [inline README.md]: "baked"?

**Comment by Adam-** [comment]
I think its good to have environment variables for the simple things everyone will want (servername, default oline password). Anything non-trivial is best left up to people running their own configurations though. I think the majority of people using this just want a quick IRC server without much fuss.

File: README.md (hunk 1 above)
Comment (SISheogorath) [inline README.md]: yes and `include` ^^

**Comment by SISheogorath** [comment]
Closing this in favor of a less complex solution.

**Pull request closed without merging**